plugins/vendor/zoom/lib/model/kZoomEventHanlder.php

```diff
@@ -55,6 +57,19 @@ public function processEvent($event)
 				if ($zoomDropFolderId)
 				{
 					$zoomClient = $this->initZoomClient($zoomVendorIntegration);
+					$hostId = $event->object->hostId;
```

Comment:
move the logic inside createZoomDropFolderFile so it will be inforced on transcripts as well